src/algebra/module/equiv.lean

```diff
@@ -652,3 +652,29 @@ rfl
 end add_comm_group
 
 end add_equiv
+
+section linear_map
+
+variables [ring R] [add_comm_group M] [module R M] (T : M →ₗ[R] M)
+
+def linear_equiv.of_invertible [invertible T] : M ≃ₗ[R] M :=
+{ to_fun := λ x, T x,
```

Comment:
I suspect you can prove this in one line using `invertible.unit` and whatever the equiv from `general_linear_group` to `linear_equiv` is called

Reply:
https://leanprover-community.github.io/mathlib_docs/linear_algebra/basic.html#linear_map.general_linear_group.to_linear_equiv is the definition.

Reply:
I think that's probably fine; although another good option (if the port allows) would be to move this `general_linear_group` stuff to its own file... I'll have some time at the airport today where I could try that.

Reply:
There's probably a lot of overlap between the things you want to state with `invertible` and the things we already have for `units` (aka `linear_map.general_linear_group`)

Reply:
Hmm... Should I move it to `linear_algebra/basic` then? Or should I wait for you to let me know if I should move `general_linear_group` to its own file?

Reply:
For now let's put it in linear_algebra/basic so we can continue to review it.

Reply:
done